zorg_grove/lcd.py

```diff
@@ -1,4 +1,5 @@
 from zorg.driver import Driver
+from zorg_i2c import I2CDriver
```

Comment:
This should actually be removed, and the reference lower down should be changed to `Driver`. The `I2CDriver` class was removed in https://github.com/zorg-framework/zorg-grove/commit/98682e46686fec32fb00ad4324542306f5694c55.